README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
 You can also [download the compiled and minified assets (CSS, JavaScript) from
 GitHub](docs/installation/installing-from-dist.md).
 
-## Browser support
+## Accessibility
 
-GOV.UK Frontend will allow you to build services that comply with the [guidance
-in the Service Manual][service-manual-browsers].
+The GOV.UK Design System team regularly tests GOV.UK Frontend with a range of users to make sure it’s as accessible as possible. You can read [the results of our most recent test for accessibility](https://design-system.service.gov.uk/accessibility/).
 
-Currently, GOV.UK Frontend officially supports the following browsers:
+However, your service will not automatically meet [level AA of WCAG 2.1](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag) if you use GOV.UK Frontend. You must still do research to check that your service meets accessibility requirements, especially if you extend or modify components.
 
-| Operating system | Browser                                | Support     |
-|----------------- |----------------------------------------|-------------|
-| Windows          | Internet Explorer 8-10                 | functional  |
-| Windows          | Internet Explorer 11                   | compliant   |
-| Windows          | Edge (latest 2 versions)               | compliant   |
-| Windows          | Google Chrome (latest 2 versions)      | compliant   |
-| Windows          | Mozilla Firefox (latest 2 versions)    | compliant   |
-| macOS            | Safari 9+                              | compliant   |
-| macOS            | Google Chrome (latest 2 versions)      | compliant   |
-| macOS            | Mozilla Firefox (latest 2 versions)    | compliant   |
-| iOS              | Safari for iOS 9.3+                    | compliant   |
-| iOS              | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Google Chrome (latest 2 versions)      | compliant   |
-| Android          | Samsung Internet (latest 2 versions)   | compliant   |
+You should also use:
 
-‘Compliant’ means that the components must look as good and function as well as
-they do in other modern browsers.
+- [the JavaScript from GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
+- [a separate stylesheet](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md) if you support Internet Explorer 8
 
-'Functional' means the components may not look perfect in the given browser, but
-must still be usable without errors and without 'looking broken'.
+Your service will not meet level AA of WCAG 2.1 if you use [compatibility mode](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/compatibility.md).
```

Comment:
Do we need to say why? Will this put people off migrating if we're not reassuring that this is still a good thing?

Reply:
Yep definitely - I've added the 'why' to the docs on the compatibility mode itself.

Reply:
I feel like there is a more positive 'migration' framed way of introducing compatibility mode, this wording assumes the reader already knows about this when it's likely they will not?

Reply:
On your positive framing/migration point, it's best for users if we keep to GOV.UK style of just the concise facts - in this case, that your service won't meet AA if you use this mode. I appreciate plain words can feel a bit 'harsh' to us as we closely read and reread the words - but it doesn't sound that way to users, promise!

But I see what you mean on the familiarity with compatibility mode. How about we add "... to use GOV.UK Frontend with old frameworks or colours" at the end to make it clearer for new users what compatibility mode is?

Reply:
> but it doesn't sound that way to users, promise!

I think it's fair to worry that our users could misunderstand this statement given what we know about people's attitude to migration.

I'm definitely not suggesting we write some flowery advertising copy, more to just ensure our users understand the feature will help them migrate to a more compliant framework, before we say something negative about it.

Through talking about this I think the bigger issue is just that we don't really have much to say about migration that's useful for our users, so it's always going to be a bit weird hearing about this for the first time in an accessibility statement.

So probably OK to leave as is and think about migration another time.